template_debian/distribution.sh

```diff
@@ -476,9 +476,22 @@ EOF
 # Uninstall Qubes Repo
 # ==============================================================================
 uninstallQubesRepo() {
-    info ' Removing Qubes build repo from sources.list.d'
+    info "Removing Qubes build repo from sources.list.d"
 
     # Lets not umount; we do that anyway when 04 exits
     umount_kill "${INSTALL_DIR}/tmp/qubes_repo"
     rm -f "${INSTALL_DIR}/etc/apt/sources.list.d/qubes-builder.list"
 }
+
+
+# ==============================================================================
+# Install Mozilla/PPA Repo
+# ==============================================================================
+installMozillaRepo() {
+    cat > "${INSTALL_DIR}/etc/apt/sources.list.d/qubes-builder.list" <<EOF
+deb https://ppa.launchpadcontent.net/mozillateam/ppa/ubuntu ${DIST_CODENAME} main
+deb-src https://ppa.launchpadcontent.net/mozillateam/ppa/ubuntu ${DIST_CODENAME} main
+EOF
+
+  cp "${TEMPLATE_CONTENT_DIR}/../keys/launchpad-ppa-for-mozilla-team.gpg" "${INSTALL_DIR}/etc/apt/trusted.gpg.d/"
```

Comment:
As Demi pointed out, this is RSA1024 key, we definitely don't want to trust such weak key in the default build. Either find something with better integrity protection (and still provided by a trusted 3rd-party), or move this into a template option (accepting that default template won't have Firefox at all, or have it via snap).

Reply:
I don't have much better than: https://launchpad.net/~mozillateam/+archive/ubuntu/ppa